### ISSUE TITLE
incorrect address prefix in 06/06/04 step 15 and 25

### ISSUE THREAD

**Issue by mkolus** [MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator#515]
# Module: 06
#### Step: 15

Address prefix should be 10.63.0.0/22.

#### Step: 25

Address prefix should be 10.62.0.0/22.

**Comment by mkolus** [comment]
No, it's not about the testing, but the vnets are created with a /22 netmask (https://github.com/MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator/blob/master/Allfiles/Labs/06/az104-06-vms-loop-template.json) and the documentation says /20. The lab still works.

**Comment by staleycyn** [comment]
This repo is for Microsoft instructors who explain /20 includes /22. The lab is working, there is no change at this time.
